client/src/config/app/locale/en/en.json

```diff
@@ -207,6 +254,58 @@
         "unauthenticated": "Login to post a comment"
       }
     },
+    "sections": {
+      "journal_info": "$t(glossary.journal_one) info",
+      "full_description": "Full description",
+      "tags_list": "Tags list",
+      "variants": "Downloadable Variants",
+      "resource_slideshow": "Resource slideshow",
+      "resource_list": "Resource list"
+    },
+    "reader": {
```

Comment:
I think, even at this stage, we should introduce some structure here and not just put everything under a reader block. Just off the top of my head, I could imagine something like:
```
"reader": {
   "actions": {
     ...keys here
   },
    "menus": {
       "settings":  {
         ...keys here
       },
       "appearance": {
         ...and so on
       }
    } 
}
```
Then we can decide later if we want to keep things under a top-level "reader" key or move some things into other categories. I'm surprised that "menus" hasn't come up yet elsewhere, but I can imagine we might later want a structure where "menus" is the top key and then "reader", "global", etc. are under it.

I'll just put [Jen's great notes](https://www.notion.so/castiron/Translations-best-practices-f9f5da36adeb4b948d6f6e3cb5c0e3e8#d0e9e065de9140a6b1007e5c1670e5df) on best practices here because I find them so helpful.